workspaces/bulk-import/packages/app/knip-report.md

```diff
@@ -0,0 +1,20 @@
+# Knip report
+
+## Unused dependencies (5)
+
+| Name                                                | Location     | Severity |
+| :-------------------------------------------------- | :----------- | :------- |
+| @red-hat-developer-hub/backstage-plugin-bulk-import | package.json | error    |
```

Comment:
Do you understand this false-positive?

The import is used in `workspaces/bulk-import/packages/app/src/App.tsx`